### ISSUE TITLE
Store.js Line 15 -> con.error()

### ISSUE THREAD

**Issue by bighappyworld** [FormidableLabs/biff#12]
You removed all the console behavior but missed a line in Store.js Line 15.

**Comment by morenoh149** [comment]
Yes I'm running into this issue as well. Would a PR that simply removes https://github.com/FormidableLabs/biff/blob/master/src/Store.js#L6-L14 be welcome? @kenwheeler

**Comment by kenwheeler** [comment]
Sure
